txnkv/txnlock/lock_resolver.go

```diff
@@ -327,16 +334,16 @@ func (lr *LockResolver) BatchResolveLocks(bo *retry.Backoffer, locks []*Lock, lo
 //    commit status.
 // 3) Send `ResolveLock` cmd to the lock's region to resolve all locks belong to
 //    the same transaction.
-func (lr *LockResolver) ResolveLocks(bo *retry.Backoffer, callerStartTS uint64, locks []*Lock) (int64, error) {
-	ttl, _, _, err := lr.resolveLocks(bo, callerStartTS, locks, false, false)
+func (lr *LockResolver) ResolveLocks(bo *retry.Backoffer, callerStartTS uint64, locks []*Lock, detail *util.ResolveLockDetail) (int64, error) {
```

Comment:
Could it be more friendly to client-go users to add a new function `ResolveLocksWithDetail` instead of modifying the original interface?